src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -125,15 +159,27 @@ public SslClientAuthenticationOptions SslOptions
 
         public bool SupportsAutomaticDecompression => false;
         public bool SupportsProxy => false;
-        public bool SupportsRedirectConfiguration => false;
+        public bool SupportsRedirectConfiguration => true;
 
         private Dictionary<string, object?>? _properties;
         public IDictionary<string, object?> Properties => _properties ??= new Dictionary<string, object?>();
 
+        protected override void Dispose(bool disposing)
```

Comment:
I think this is also unnecessary since there's no check in `SendAsync`.
Also base `Dispose` doesn't do anything:
https://github.com/kjpou1/runtime/blob/wasm-http-redirect/src/libraries/System.Net.Http/src/System/Net/Http/HttpMessageHandler.cs#L31

I think both `_disposed` and `_handlerStarted` can be removed as well as `Dispose`. Unless I'm overlooking something.

Reply:
Removed from code.  Thanks for verifying.